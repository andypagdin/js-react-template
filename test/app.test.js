import React from 'react'
import App from '../src/components/app'
import renderer from 'react-test-renderer'

it('it renders the app component correctly', () => {
    const component = renderer.create(<App />).toJSON()
    expect(component).toMatchSnapshot()
})
