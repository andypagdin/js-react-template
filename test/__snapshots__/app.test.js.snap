// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`it renders the app component correctly 1`] = `
<h1>
  Home
</h1>
`;